test/utility/misc.c

```diff
@@ -410,7 +410,8 @@ testutil_wiredtiger_open(TEST_OPTS *opts, const char *home, const char *config,
     if (opts->tiered_storage)
         testutil_check(__wt_snprintf(tiered_ext_cfg, sizeof(tiered_ext_cfg),
           TESTUTIL_ENV_CONFIG_TIERED_EXT TESTUTIL_ENV_CONFIG_TIERED, opts->build_dir,
-          opts->tiered_storage_source, opts->tiered_storage_source, benchmarkrun ? 0 : 2,
+          opts->tiered_storage_source, opts->tiered_storage_source, opts->delay_ms,
+          opts->force_delay, opts->error_ms, opts->force_error, benchmarkrun ? 0 : 2,
```

Comment:
This should be alphabetized wherever the format is specified so that `error_ms` is not in between the two `force` entries.

Reply:
Fixed in 9e36aaa699dcd933cf4d399d2dea888ed80ae3ba